tests/CustomElements/v1/js/upgrade.js

```diff
@@ -61,4 +61,32 @@ suite('Upgrades', function() {
     assert.instanceOf(e2, X2);
   });
 
+  suite('customised build-in elements', function() {
```

Comment:
I wasn't sure the best place to put these tests. Upgrade *seemed* relevant, but the tests inside it at first glance didn't correlate well with the tests I wanted to write. I just put them here for now. Can easily change later.